Member-Travel-Fund.md

```diff
@@ -0,0 +1,40 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation to travel and spread knowledge about the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be Individual Members of the Node.js Foundation.
+* Candidates must be presenting one of the following presentations:
```

Comment:
We should discuss this a little more.

I'm basing this a bit on the success we had with the io.js Roadmap, which was translated in to like 11 languages and given at I don't even know how many meetups. One of the things we got out of that was collaborative iteration of the deck and an "approved" resource that encouraged localization.

I gave this Foundation talk at FresNode last week and there was plenty of "personal flair" in the way I delivered it even if the deck lacked personality. However, I do like talks that are more personal, and add a bit about the persons involvement, and I certainly touched on that during my delivery but didn't have anything extra in the deck to support it visually.